react/components/EditorContainer/StoreEditor/Store/PWAForm/index.tsx

```diff
@@ -292,16 +250,95 @@ const PWAForm: React.FunctionComponent<Props> = ({
           </div>
         </div>
       )}
-      <div className="pt6">
-        <Toggle
-          label={intl.formatMessage(messages.disablePrompt)}
-          checked={!settings.disablePrompt}
-          disabled={submitting}
-          onChange={() =>
-            setSettings({ ...settings, disablePrompt: !settings.disablePrompt })
-          }
-        />
-      </div>
+      {!showSpecific ? (
+        <>
+          <div className="pv7">
+            <div className="w-100 bb b--muted-4" />
+          </div>
+          <div
+            className="link pointer c-muted-1"
+            onClick={() => setShowSpecific(true)}
+          >
+            <span className="pr4">
+              <FormattedMessage id="admin/pages.editor.store.settings.pwa.app-settings" />
```

Comment:
why not always render this and use the `showSpecific` only to render the hidden form inputs?